src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -951,7 +951,24 @@ public with sharing class CRLP_RollupUI_SVC {
 
     }
 
-    
+    /***
+    * @description Checks if the given field name is not system and einstein insights fields
+    * @param dfr the DescribeFieldResult of the field
+    * @return Boolean
+    */
+    private static Boolean excludeSystemAndEinsteinInsightsFields(Schema.DescribeFieldResult dfr) {
```

Comment:
What would you think of refactoring as
`private static Boolean excludeField(Schema.DescribeFieldResult dfr)` and have the method reference private var in the class such as
```
private static final Set<String> fieldsToExclude = new Set<String>{
    'topinsightid',
    'activitymetric',
    'opportunityscore',
    'activitymetricrollup',
    'npo02__system_custom_naming__c',
    'npe01__system_accounttype__c',
    'npe01__systemisindividual__c',
    'npe01__systemaccountprocessor__c',
    'npo02__systemhouseholdprocessor__c',
    'npo02__systemhouseholdcontactroleprocessor__c'
};
```

and then reference that in the method?

Reply:
That is fine with me, are you sure those are the only system fields you get in CRLP UI?

Reply:
Yes. I searched through all the packages.

Reply:
@force2b , I have added activitymetric as well as actvitymetricid, activitymetricrollup and activitymetricrollupid. I am not sure whether it appends id all the time or not.